docs/downloads/automation-library/auto_pr_description.cm

```diff
@@ -0,0 +1,14 @@
+triggers:
```

Comment:
Should this also have a trigger to only run when the PR is created or has new commits pushed?